src/app/shared/module/poe/component/item-frame-header/item-frame-header.component.html

```diff
@@ -5,12 +5,12 @@
     </app-item-frame-query>
   </div>
   <div class="name" *ngIf="item.typeId">
-    <app-item-frame-query
-      [(property)]="queryItem.typeId"
-      [value]="item.typeId"
-      [disabled]="!queryable"
-    >
-      <span>{{ item.typeId | baseItemType: language }}</span>
+    <app-item-frame-query [(property)]="queryItem.typeId" [value]="item.typeId" [disabled]="!queryable">
+      <ng-container [ngSwitch]="getNameSpanNum(item)">
+        <span *ngSwitchDefault>{{ item.typeId | baseItemType: language }}</span>
```

Comment:
i believe best practice is default case to be at the end of the switch. also not a big deal.

Reply:
Moved to the bottom in c77b9ceb0fcde8cb7a50c2841ba465e01f135afc